app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/package.scala

```diff
@@ -30,6 +30,8 @@ package object models {
 
   // formats
 
+  type ClaimsAnswer = NonEmptyList[Claim]
```

Comment:
Sorry @gabor-dorcsinecz please can you put this to `answers` package for the consistency purposes

Reply:
Done

Reply:
You are cheating :D